chef_master/source/data_collection_ha.rst

```diff
@@ -0,0 +1,40 @@
+Data Collection with a Chef High Availability Cluster
+========================================================
+`[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/data_collection_ha.rst>`__
+
+.. important:: Previous step: :doc:`Setup Data Collection </data_collection>`
```

Comment:
Maybe a bit pedantic, but I think that bookending such a short guide with both a warning and an important note makes it look a bit cluttered - this one seems like it's less worthy of a note and could maybe be included in the introductory paragraph.

Reply:
I'm going to keep it, because it is consistent with the other pages.